docroot/themes/humsci/humsci_basic/templates/html.html.twig

```diff
@@ -30,6 +30,7 @@
  * @ingroup themeable
  */
 #}
+
```

Comment:
I was testing things in this file. I can remove this space. :smile:

Reply:
great!